ui/job-view/pushes/FuzzyJobFinder.jsx

```diff
@@ -0,0 +1,276 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  InputGroup,
+  InputGroupAddon,
+  Input,
+} from 'reactstrap';
+import Fuse from 'fuse.js';
+
+import PushModel from '../../models/push';
+import { withNotifications } from '../../shared/context/Notifications';
+import { formatTaskclusterError } from '../../helpers/errorMessage';
+import { sortAlphaNum } from '../../helpers/sort';
+
+class FuzzyJobFinder extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      modal: this.props.modal,
```

Comment:
Do we need ``modal`` in the state?  We already have it in the ``props``?

Reply:
Actually, maybe we don't need ``modal`` as a prop either.  I don't think it provides anything useful, does it?  I might be missing something, though.  :)

Reply:
I think I copied it from some example somewhere and misused it. Gone, and everything seems to work still.